### PR TITLE
Remove empty transaction file exception message

### PR DESCRIPTION
**Pull request by xiaoge26**
Solve issue #93

### CODE REVIEW

**Review by tyuyang — CHANGES_REQUESTED** [review]
Oops sorry if I was not clear with the issue. Let's say a user creates a new account and then logs out without making a transaction. This will cause the transaction save file to be empty, which will then throw the SaveFileIsEmptyException and then show an error message (line 32 and 33 in TransactionList.java). The issue is that there is no need to show the error message in the first place since it is normal for there to be no transactions in the transaction save file

**Comment by tyuyang** [comment]
Sorry for the trouble

File: src/main/java/seedu/bankwithus/BankWithUs.java [hunk 1]
```diff
@@ -59,8 +59,7 @@ public BankWithUs(String accountsFilePath, String transactionsFilePath) throws I
         try {
             transactionList = new TransactionList(storage.loadTransactions());
         } catch (FileNotFoundException e) {
-            // If transactionFile not created
-            ui.showFileNotFoundError();
+            // If transactionFile not created, show nothing
```
[the remark this answers is not in the document]
Comment (xiaoge26) [reply src/main/java/seedu/bankwithus/BankWithUs.java]: Ok thanks for pointing out that!

**Review by tyuyang — APPROVED** [review]
Yep looks good

**Pull request merged**